project/src/main/webapp/index.html

```diff
@@ -33,18 +33,20 @@
       <br />
       <label for="price-level">Price</label>
       <select name="price-level" id="price-level">
-        <option value="low">Low</option>
-        <option value="medium">Medium</option>
-        <option value="high">High</option>
+        <option value="0">Free</option>
```

Comment:
er .... is this a real option?

Reply:
we have this here right now just to match the value range of `price_level` on the Places API (goes from 0-4, with 0 being "free", 4 being "very expensive"). We are planning to give the user options low, medium, high, where low corresponds to to 0 and 1, medium to 2, and high to 3 and 4

Reply:
I'm sure price_level 0 won't apply to restaurants but we were going to group it in with the "low" price range just to be safe

Reply:
got it - sounds good 👍